code/src/moviedetails.css

```diff
@@ -0,0 +1,67 @@
+.background-img {
+  width: 100vw;
+  height: 100vh;
+  background-size: cover;
+  background-position: center;
+  background-attachment: fixed;
+}
+
+.movie-details {
+  position: fixed;
+  left: 5%;
+  bottom: 40px;
+  margin: 0 auto;
+  padding: 15px;
+  display: flex;
+  align-items: flex-end;
+}
+
+.movie-title {
+  font-size: 22px;
+  font-weight: 550;
+  padding: 5px 0 0 15px;
+}
+
+.rating-text {
+  padding-left: 5px;
+  font-size: 18px;
+  font-weight: 400;
+  color: #df2a2a;
+}
+
+.back-text {
+  text-decoration: none;
+  color: #ffffff;
```

Comment:
You don't need to set the color to white here since the color white is already declared to the h2-tags.